src/Sensor/SensorGps.cpp

```diff
@@ -46,6 +46,9 @@ void SensorGps::handle() {
 
         // Calculate XY Acceleration
         float horizontalSpeed = _gps->getGroundSpeed() / MILIMETERS_IN_METERS;
+        if(_speedCallback){
+        _speedCallback(horizontalSpeed); 
```

Comment:
Just a minor style thing, can you indent the body of the if statement for clarity?